Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-listSearchIndexes.txt

MongoDB\\Collection::listSearchIndexes()
========================================

.. meta::

>    :description: Retrieve index information for search indexes in a collection using `listSearchIndexes()` with various customizable options.

Reviewer: ```suggestion
   :description: Retrieve index information for all search indexes in a collection with the MongoDB PHP Library, with various customizable options.
```
